MsCorePkg/Library/SecureBootKeyStoreLib/SecureBootKeyStoreLib.c

```diff
@@ -0,0 +1,100 @@
+/** @file SecureBootKeyStoreLib.c
+
+
+  Copyright (C) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <UefiSecureBoot.h>
+#include <Guid/ImageAuthentication.h>
+#include <Library/SecureBootVariableLib.h>
+
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/PcdLib.h>
+
+#define PLATFORM_SECURE_BOOT_KEY_COUNT  2

```

Comment:
It seems this and `gSecureBootPayloadCount` can be dropped and initialize the array with the fixed number of items for simplicity:

`SECURE_BOOT_PAYLOAD_INFO  mSecureBootPayload[] = {`...

Then, in the C code, just use `ARRAY_SIZE()` to get the number of elements:

```c
  *KeyCount = ARRAY_SIZE (mSecureBootPayload);
```

Reply:
Updated! Thank you.